docs/en/reference/migrating.md

```diff
@@ -0,0 +1,41 @@
+
+Once you've copied the workspace folders containing your notes from your old computer and placed them on the new machine, you can add each workspace and its files to your new Zettlr install simply by going to the `File > Open Workspace …` menu option and selecting that workspace folder. Note that this is the same process you would use you if were adding any other folder to Zettlr. The workspace folder, including the notes it contains, will immediately be populated into your new Zettlr install. The connections between notes will also be retained, though if you have links to notes that live in different workspaces, you'll have to make sure you've likewise opened those workspace folders and their contents within your new Zettler install.
+
+Any files or notes that aren't part of a workspace folder can also be opened in your new Zettlr install once you've moved them to your new machine, just as you would use Zettlr to open any other file.
+
+##  Copying Your Application Settings to the New Machine
+
+If you've customized Zettlr at all and/or you want to retain the various stats about your notes, there are a few configuration files that you may want to back up and copy over to your new machine. This data is located in the _application data path_ whose location depends on your Operating System:
+
+* **Windows:** `C:\Users\<your username>\AppData\Roaming\Zettlr` (note that AppData is a hidden folder, so you might want to change the Explorer settings so it shows up)
+* **macOS:** `/Users/<your username>/Library/Application Support/Zettlr` (open a Finder window, hold down "Alt" while opening the "Go"-menu, and click on the appearing "Library" entry)
+* **Linux:** `/home/<your username>/.config/Zettlr` (note that `.config` is a hidden folder, so you might want to use a command line to open it)
+
+Once you've located this folder, save a copy of the following files and subfolders within it to move to your new machine:
+
+* `stats.json` (your writing statistics)
+* `custom.css` (your custom CSS, if any)
+* `tags.json` (your colour-coded tags)
+* `targets.json` (your writing targets)
+* `user.dic` (your own custom dictionary)
+* `defaults` (default settings for document import and export)
+* `snippets` (any snippets you've set up for auto-filling note content)
+
+Once you've installed Zettlr on your new computer and launched it for the first time, the software will create a new application path folder (see above). When you're ready to import the application settings to your new computer, close Zettlr and copy the backups of the files and subfolders you just made into the newly created application path folder. If you're prompted as to whether it's okay to replace existing files, you can say "okay." You've now copied over the application settings from your old computer.
```

Comment:
I would move that one up, below the section with moving files, so that a user sees "Ah, first I need to copy files. Then I need to install Zettlr, launch it, and close it again, and then copy those files".

Therefore I would also reword this to something more explicit with regard to what users have to do: "Install Zettlr and launch it. This will create the configuration directory into which you can then put your files. Then, quit the app again, and copy all the files you wish to keep from your old install into the new configuration directory."

Maybe additionally, we want to add some note of caution: "Zettlr will automatically remove the paths to the workspaces of your old setup, since they are not valid anymore. So when you start Zettlr after copying the files, you will be greeted with an empty window. Now you can open the workspaces again."

(Feel free to adapt the wording, I'm typing this on my phone)

Reply:
I've commented above – my problem was that when I wrote the first comment in my mind it was like "config should not be moved" and while I was writing this last comment my mind suddenly turned to "actually it should be fine" :D